keps/sig-network/3866-nftables-proxy/README.md

```diff
@@ -1678,19 +1642,14 @@ are missing a bunch of machinery and tooling and can't do that now.
 
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
-<!--
-Even if applying deprecation policies, they may still surprise some users.
--->
+The new backend is not 100% compatible with the `iptables` backend.
```

Comment:
> Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?

I'm fine with the test not being executed now, but can you briefly describe the test that will be run prior beta graduation? [and update the details based on results later (probably after freeze)]?

Reply:
this was answered in the "feature enablement/disablement" section but I'll mention it here too